src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -52,13 +53,13 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
     public void execute_someFieldsSpecifiedUnfilteredList_success() {
         Index indexLastAnimal = Index.fromOneBased(model.getFilteredAnimalList().size());
         Animal lastAnimal = model.getFilteredAnimalList().get(indexLastAnimal.getZeroBased());
-
+        System.out.println(lastAnimal.getId());
```

Comment:
I assume this was for debugging? Suggest to remove or if need be, change to a logging statement.

Reply:
Line removed